docs/model-monitoring/index.md

```diff
@@ -126,4 +127,11 @@ See an example of batch input in the [Serving pre-trained ML/DL models](../tutor
 ## Alerts and notifications
 
 You can set up {ref}`alerts` to inform you about suspected and detected issues in the model monitoring functions. 
-And you can use {ref}`notifications` to notify about the status of runs and pipelines.
\ No newline at end of file
+And you can use {ref}`notifications` to notify about the status of runs and pipelines.
+
+## Scale limitations
+
+When ramping up the scale of your model monitoring, take note of these limitations. Each limit here assumes you have only large, medium, or small projects. When working with a combination, use these limits proportionally to to your projects.
+- Up to 20 large projects (MEPs per project between 1k and 5k)
+- Up to 100 medium projects (100 < MEPs < 1k)
+- Up to 200 small projects (MEPs < 100)
```

Comment:
should explicitly state model endpoints, MEP is not really used in the documentation